internal/config/config.go

```diff
@@ -15,6 +16,12 @@ type Config struct {
 	Hooks map[string]*Hook
 }
 
+type Remote struct {
+	URL     string   `mapstructure:"url"`
```

Comment:
I think `url` option is too general. Maybe `git`?

Reply:
Would `git_url` or `git_repo` be an acceptable option? In #155 the posted example uses `repo` which may be another good alternative.

Reply:
I like `git_url`, it perfectly explains the meaning of an option.